src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -271,6 +270,16 @@ public void addConfiguration(InstanceConfiguration configuration) {
     return result;
   }
 
+  /**
+   * Choose semi randomly config from list of available configs.
```

Comment:
This documentation seems misleading, as this is round robin not random.

Reply:
Its semi round robin.. cause it shares ID with all configs... so its random round robin :)

At current moment 99% of instances ends up always with first config..
Here all configs should be used rather uniformly..

I can make it full random.. 
But then there is huge problem with testing it..

Reply:
I can change javadoc if you want..

Reply:
I think some additional clarification along those lines would be helpful. Thanks.